common.blocks/form-field/form-field.spec.js

```diff
@@ -0,0 +1,29 @@
+modules.define('spec',
+    ['form-field', 'BEMHTML', 'i-bem__dom'],
+    function(provide, FormField, BEMHTML, BEMDOM) {
+
+    describe('form fields manipulating', function() {
+
+        var formField;
+
+        beforeEach(function() {
+            formField = buildFormField();
+        });
+
+        it.skip('should can get tab-index', function() {
+            formField.getIndex().should.be.eq(1);
+        });
+
+        it.skip('should can set tab-index', function() {
+            formField.getIndex().should.be.eq(1);
+            formField.setIndex(-1).should.be.eq(-1);
```

Comment:
Okay, what if I change form from 2 places, how to know what index I should set for my field?
Isn't better to use `form.addField(formField, {after: anotherField})` ?